external-crates/move/move-compiler/src/parser/syntax.rs

```diff
@@ -2435,7 +2435,12 @@ fn parse_constant_decl(
     }
     consume_token(context.tokens, Tok::Const)?;
     let name = ConstantName(parse_identifier(context)?);
-    consume_token(context.tokens, Tok::Colon)?;
+    if consume_token(context.tokens, Tok::Colon).is_err() {
```

Comment:
```suggestion
    if context.tokens.peek() != Tok::Colon {
```

Reply:
consume_token is just a helper to generate an error, no need to do that just to through it away